scio-avro/src/main/scala/com/spotify/scio/avro/taps.scala

```diff
@@ -163,6 +166,6 @@ final case class AvroTaps(self: Taps) {
     val avroT = AvroType[T]
 
     import scala.concurrent.ExecutionContext.Implicits.global
-    avroGenericFile(path, avroT.schema, params).map(_.map(avroT.fromGenericRecord))
```

Comment:
should we add scalafix rules for `avroGenericFile` to `avroFile` transform?

Reply:
You're not looking at the full diff. `avroGenericFile` was created but then reverted by using overload. `avroFile` was the API and is kept after the changes.